nemo/collections/nlp/models/language_modeling/megatron_gpt_peft_models.py

```diff
@@ -151,21 +190,53 @@ def init_peft_modules(self):
         Randomly initialize the peft params and add them to the appropriate modules.
         """
         assert len(self.peft_name_keys) > 0, "peft_name_keys have not been set no PEFT modules will be added"
+        assert not self.mcore_gpt or hasattr(
+            self, 'name_key_to_mcore_mixins'
+        ), "some of the PEFT modules are not supported in megatron core mode yet."
```

Comment:
could we print self here so that the error message says which module it is?

Reply:
done.